_maps/configs/syndicate_hardliner_phantom.json

```
@@ -0,0 +1,42 @@
+{
+	"faction": "/datum/faction/hardliners",
```

```suggestion
	"faction": "/datum/faction/syndicate/hardliners",
```

Your faction datum won't allow access to general coalition gear otherwise.